fitstack/signal.py

```diff
@@ -109,7 +51,15 @@ def __init__(
         self._aliases = aliases if aliases is not None else {}
 
     @classmethod
-    def load_from_stackfiles(cls, pattern: str, **kwargs):
+    def load_from_stackfiles(
+        cls,
+        pattern: str,
+        pol: List[str] = None,
+        weight: np.ndarray = None,
```

Comment:
Ah great. This has been changed from the other PR. I was finding things were breaking on cedar as the numpy wasn't new enough.

Reply:
Yeah I was running into the same issue.